astropy/coordinates/builtin_frames/lsr.py

```diff
@@ -14,7 +14,7 @@
 from .baseradec import BaseRADecFrame, doc_components as doc_components_radec
 from .icrs import ICRS
 from .galactic import Galactic
-from .fk4 import FK4
+from .fk4 import FK4  # noqa
```

Comment:
@adrn, is this import really necessary?

Reply:
No, that is vestigial and could be removed now
```suggestion
```

Reply:
This PR is going nowhere, so I removed it directly in 6c15aa65851dc31e8cac843bd16e83178c8b50bf . Thanks for the response!